<details><summary>Linter check found the following problems:</summary>
Traceback (most recent call last):
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/run.py", line 208, in main
    return_code, report = execute_linter(
                          ^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/run.py", line 180, in execute_linter
    result = linter.lint(recipes, subdir, variant_config_files, exclusive_config_files, fix_flag)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/lint/__init__.py", line 725, in lint
    msgs = self.lint_file(
           ^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/lint/__init__.py", line 831, in lint_file
    self.lint_recipe(
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/lint/__init__.py", line 891, in lint_recipe
    res = self.check_instances[check].run(recipe, fix)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/lint/__init__.py", line 294, in run
    self.check_recipe(recipe)
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/lint/check_syntax.py", line 36, in check_recipe
    has_constraints = constraints.search(spec)
                      ^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: expected string or bytes-like object, got 'NoneType'

ERROR conda.cli.main_run:execute(125): `conda run conda-lint /tmp/abs_56ngon3e0h/clone` failed. (See above for error)
</details>

--- a/.gitignore
+++ b/.gitignore
@@ -6,6 +6,7 @@
 # Ignore all files and folders in root
 *
 !/conda-forge.yml
+!/abs.yaml
 
 # Don't ignore any files/folders if the parent folder is 'un-ignored'
 # This also avoids warnings when adding an already-checked file with an ignored parent.

--- a/abs.yaml
+++ b/abs.yaml
@@ -1,0 +1,6 @@
+build_env_vars:
+  ANACONDA_ROCKET_ENABLE_PY313 : yes
+
+upload_channels:
+  - sfe1ed40
+

--- a/recipe/build.sh
+++ b/recipe/build.sh
@@ -13,4 +13,6 @@ fi
 autoreconf -if
 ./configure --prefix=$PREFIX
 make
+# Enable tests
+make check
 make install

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -20,7 +20,7 @@ build:
 requirements:
   build:
     - {{ compiler('c') }}
-    - {{ stdlib("c") }}
+    #- {{ stdlib("c") }}
     - {{ compiler('cxx') }}
     - make       # [unix]
     - libtool    # [unix]

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -1,33 +1,34 @@
 {% set name = "libassuan" %}
 {% set version = "3.0.1" %}
-{% set sha256 = "c8f0f42e6103dea4b1a6a483cb556654e97302c7465308f58363778f95f194b1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
   url: https://www.gnupg.org/ftp/gcrypt/{{ name }}/{{ name }}-{{ version }}.tar.bz2
-  sha256: {{ sha256 }}
+  sha256: c8f0f42e6103dea4b1a6a483cb556654e97302c7465308f58363778f95f194b1
 
 build:
   number: 0
   skip: true  # [win]
+  # libgpg-error does not exist on s390x
+  skip: True  # [s390x]
   run_exports:
     - {{ pin_subpackage('libassuan', max_pin='x') }}
 
 requirements:
   build:
     - {{ compiler('c') }}
-    #- {{ stdlib("c") }}
     - {{ compiler('cxx') }}
     - make       # [unix]
     - libtool    # [unix]
     - automake   # [unix]
     - gnuconfig  # [unix]
   host:
-    - libgpg-error
+    # https://github.com/gpg/libassuan/blob/libassuan-3.0.1/configure.ac#L402
+    # but avaliable only version 1.42 in the main channel.
+    - libgpg-error ==1.42
 
 test:
   commands:
@@ -41,8 +42,13 @@ about:
   description: |
     Libassuan is a small library implementing the so-called Assuan protocol.
     This protocol is used for IPC between most newer GnuPG components.
+  doc_url: https://www.gnupg.org/software/libassuan/index.html
+  dev_url: https://github.com/gpg/libassuan
 
 extra:
   recipe-maintainers:
     - mariusvniekerk
     - xhochy
+  skip-lints:
+    # Added make check in the build.sh
+    - missing_tests

